interfaces/Portalicious/tailwind.config.ts

```diff
@@ -138,15 +139,6 @@ export default {
         },
         { values: theme('typographySizes') },
       );
-      addUtilities({
```

Comment:
<img width="694" alt="image" src="https://github.com/user-attachments/assets/843956c5-76cd-4a03-9448-1a3ca215f237">

My local `npm run typecheck` is now complaining.. :'( @aberonni

Reply:
My bad! I will create a new PR that also includes a typecheck in CI :D